test/test-v8debugapi.ts

```diff
@@ -218,7 +221,8 @@ describeFn('debugapi selection on Node >=10', () => {
 describe('v8debugapi', () => {
   const config: ResolvedDebugAgentConfig = extend(
       {}, defaultConfig, {workingDirectory: __dirname, forceNewAgent_: true});
-  // TODO(dominickramer): It appears `logLevel` is a typo and should be `level`.  However,
+  // TODO(dominickramer): It appears `logLevel` is a typo and should be `level`.
+  // However,
   //       with this change, the tests fail.  Resolve this.
   const logger =
       new common.logger({levelLevel: config.logLevel} as {} as LoggerOptions);
```

Comment:
Not this PR, but is `levelLevel` really a correct field name?

Reply:
I think this should be addressed in a follow-up PR.

Reply:
Sounds good. I'm just wondering how tests work when the log level is set incorrectly.